### PR TITLE
Support for conda-based installation (as well as using conda env).

### PR DESCRIPTION
**Pull request by tesfaldet**

### CODE REVIEW

**Comment by tesfaldet** [comment]
Tried a practice build in the conda env and everything worked fine. index.html was created and there were no package issues.

**Review by btravouillon — APPROVED** [review]
Looks good to me. 👍

File: docs/environment.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  - conda-forge
+  - defaults
+dependencies:
+  - sphinx>=4.0.2
+  - recommonmark>=0.7.1
+  - myst-parser>=0.14.0
+  - sphinx-copybutton>=0.3.1
+  - sphinx-prompt>=1.4.0
+  - sphinx_rtd_theme>=0.5.2
+  - pip=22.0
```
Comment (satyaog) [inline docs/environment.yaml:12]: should this be
```suggestion
  - pip>=22.0
```

to allow installation of the most up to date version of pip?

**Comment by tesfaldet** [comment]
Integrated suggested change of pip requirement by @satyaog

**Pull request merged**